### ISSUE TITLE
Accept publication metadata during configuration

### ISSUE THREAD

**Issue by respinos** [mlibrary/cozy-sun-bear#50]
Reader can be provided publication metadata to augment/override publication metadata present in the OPF `<metadata>` block.

- [x] pass publication metadata to the `cozy.reader` initialization
- [x] add a method to set publication metadata given a reader instance

Accepted metadata:

- [ ] identifier
- [ ] title
- [ ] date
- [ ] creator (array)
- [ ] publisher
- [ ] rights
- [ ] doi
- [ ] download_link

(see: http://www.idpf.org/epub/30/spec/epub30-publications.html#sec-metadata-elem)

**Comment by respinos** [comment]
Should *contributor* be a complicated array of objects with names and roles, or should we pass in arrays by role? (authors, editors, etc)?

The EPUB spec gets complicated here: http://www.idpf.org/epub/30/spec/epub30-publications.html#role

```xml
<metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
    …
    <dc:creator id="creator">Haruki Murakami</dc:creator>
    <meta refines="#creator" property="role" scheme="marc:relators" id="role">aut</meta>
    …
</metadata>```

**Comment by respinos** [comment]
For citations, assuming this metadata has been pre-parsed if passed to the reader instance --- e.g.

```
creator: [
  'Alex Mock'
],
editor: [
  'Felix Faux'
]
```

Closed by PR#51
